docs/operations/debug.md

```diff
@@ -0,0 +1,59 @@
+# Debug
+This page contains instructions on how to debug Kubevirt.
+
+This is useful to both Kubevirt developers and advanced users that would like to gain deep understanding on what's
+happening behind the scenes.
+
+## Log Verbosity
+Kubevirt produces a lot of logging throughout its codebase. Some log entries have a verbosity level defined to them.
+The verbosity level that's defined for a log entry determines the minimum verbosity level in order to expose the
+log entry.
+
+In code, the log entry looks similar to: `log.Log.V(verbosity).Infof("...")` while `verbosity` is the minimum verbosity
+level for this entry.
+
+For example, if the log verbosity for some log entry is `3`, then the log would be exposed only if the log verbosity
+is defined to be equal or greater than `3`, or else it would be filtered out.
+
+Currently, log verbosity can be defined per-component or per-node. The most updated API is detailed [here](https://kubevirt.io/api-reference/master/definitions.html#_v1_logverbosity).
+
+### Setting verbosity per Kubevirt component
+One way of raising log verbosity is to manually determine it for the different components in `KubevirtCR`:
```

Comment:
I haven't seen this way of referring to the KubeVirt CR, is this in use elsewhere? (If not, it might be better to break it out as the `KubeVirt` `CustomResource` or `KubeVirt` CR)

Reply:
Done, thanks!